InOut/CMakeLists.txt

```diff
@@ -113,15 +109,14 @@ if(USE_PULSEAUDIO)
     make_plugin(rtpulse rtpulse.c ${PULSEAUDIO_LIBRARY} ${PULSESIMPLE_LIBRARY})
 endif()
 
-if(PORTAUDIO_FOUND OR USE_VCPKG)
+if(USE_PORTAUDIO AND ((TARGET portaudio_static) OR (TARGET portaudio)))
```

Comment:
Is this the recommended way to check if a target exists? Isn't it convention to set `HAVE_X` package?

Reply:
Yeah, but the vcpkg portaudio find module doesn't set a variable like that

Reply:
Yeah but if you have `find_package(portaudio CONFIG REQUIRED)` the whole build fails if it's not found. It seems redundant to check here in this kind of awkward way.

Reply:
> find_package(portaudio CONFIG REQUIRED) the whole build fails if it's not found

That would make sense to me too. But @kunstmusik was saying that we want to turn variables like `USE_PORTAUDIO` off if the dependencies aren't found. I would have done this with `check_deps`, but I couldn't figure out how to get the `(TARGET portaudio)` conditions to pass through as function arguments.

Reply:
Added `check_deps` for clarity

Reply:
Ah right. Yeah I see what you mean. There is a flag `FAIL_NOT_FOUND` or something that we use to fail the build if a dependency is missing. We could possibly use it in that case.  Maybe we will need to explicitly check for the targets so